backend/app/utils/api_response.py

```diff
@@ -0,0 +1,47 @@
+"""API response classes."""
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
+class ApiResponse(ABC):
+    """Abstract base class for API responses."""
+
```

Comment:
Move api_response.py from utils folder to resources folder. as what you defined there isn't a utility class.